modal/functions.py

```diff
@@ -1260,7 +1263,7 @@ async def remote_gen(self, *args, **kwargs) -> AsyncGenerator[Any, None]:
         async for item in self._call_generator(args, kwargs):  # type: ignore
             yield item
 
-    def call(self, *args, **kwargs) -> Awaitable[Any]:
+    def call(self, *args, **kwargs) -> Awaitable[Any]:  # type: ignore[return]
```

Comment:
Why was this added? It also won't get transferred to the generated type stubs in the pyi-file so it shouldn't make a difference

Reply:
`mypy` sees this function as returning `None`. This doesn't raise error when we type check against stubs but does so when we type check the implementation (which this PR is suggesting we do).

To avoid confusion I'll close out this PR for now.

Reply:
Ah, I see now that it's the deprecated/non-function `call` method you're changing. We might as well just remove the return type (or change it to `None`). No need to close the PR, I think it's valuable and we'll merge it as soon as we've ironed out the questionmarks :)